components/kubeconfig-service/README.md

```diff
@@ -0,0 +1,46 @@
+# (OIDC) Kubeconfig Service
+
+## Overview
+The Kubeconfig Service is a single purpose [REST](https://en.wikipedia.org/wiki/Representational_state_transfer) based API. It is designed to retrieve a kubeconfig file from an SKR cluster, and change the default authentication mechanism (token), to [kube-login](https://github.com/int128/kubelogin)
+
+## Configuration
+The application uses the following Environmental Variables for configuration:
+
+| Parameter | Required | Description | Default value |
+| :---: | :--- | :--- | :---: | 
+| **PORT_SERVICE** | No | Port used by the application. | `8000` |
+| **PORT_HEALTH** | No | Port used by the application for health check. | `9000` |
+| **GRAPHQL_URL** | Yes | Full URL of the chosen [GraphQL](https://graphql.org/learn/) service. | `http://127.0.0.1:3000/graphql` |
+| **OIDC_KUBECONFIG_ISSUER_URL** | Yes | Full URL of the chosen OIDC Issuer instance used for the Kubeconfig generation. | None |
+| **OIDC_KUBECONFIG_CLIENT_ID** | Yes | ClientID for the chosen OIDC Issuer used for the Kubeconfig generation. | None |
+| **OIDC_KUBECONFIG_CLIENT_SECRET** | Yes | Client secret for the chosen OIDC Issuer used for the Kubeconfig generation. | None |
+| **OIDC_ISSUER_URL** | Yes | Full URL of the chosen OIDC Issuer instance. | None |
+| **OIDC_CLIENT_ID** | Yes | ClientID for the chosen OIDC Issuer. | `""` |
+| **OIDC_CA** | No | CA certificate file path. | None |
+| **OIDC_CLAIM_USERNAME** | No | Identifier of the user in JWT claim. | `email` |
+| **OIDC_CLAIM_GROUPS** | No | Identifier of groups in JWT claim. | `groups` |
+| **OIDC_USERNAME_PREFIX** | No | If provided, all users will be prefixed with this value to prevent conflicts with other authentication strategies. | None |
+| **OIDC_GROUPS_PREFIX** | No | If provided, all groups will be prefixed with this value to prevent conflicts with other authentication strategies. | None |
```

Comment:
```suggestion
| **OIDC_GROUPS_PREFIX** | No | If provided, all groups are prefixed with this value to prevent conflicts with other authentication strategies. | None |
```